support/fixtures/factories/file.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const factory = require('rosie').Factory;
+const faker = require('faker');
+
+factory
+  .define('file')
+  .option('fromServer', false)
+  .attrs({
+    createdAt: () => faker.date.past().toISOString(),
+    contentType: () => faker.system.mimeType(),
+    description: () => faker.hacker.phrase(),
+    filename: () => `${faker.hacker.noun()}.${faker.system.commonFileExt()}`,
+    id: () => faker.random.number(),
```

Comment:
Looks like this should be a UUID.
https://github.com/ndustrialio/files-api/blob/master/models/file.js#L12